app/src/sharedTest/java/org/oppia/app/utility/ProgressMatcher.java

```diff
@@ -0,0 +1,37 @@
+package org.oppia.app.utility;
+
+import androidx.annotation.CheckResult;
+import androidx.test.espresso.matcher.BoundedMatcher;
+import android.view.View;
+import android.widget.ProgressBar;
+import org.hamcrest.Description;
+
+
```

Comment:
Convert this file to Kotlin. Remove this blank line above. Also, as mentioned earlier in one of the files, always write Documentation for new classes. Also, In this particular case please mention the reference URL also.

Check `DrawableMatcher` for reference.

Reply:
done